### ISSUE TITLE
Dismiss on Lifecycle.Event.ON_PAUSE

### ISSUE THREAD

**Issue by znakeeye** [skydoves/Balloon#77]
**Is your feature request related to a problem?**

When my lifecycle owner is paused, the tooltip owner is still displayed when resumed. This poses a problem if `onResume()` itself shows the tooltip - causing multiple clones of my tooltip to be displayed when toggling resumed/paused state.

**Describe the solution you'd like:**

Not sure if this should be considered a bug or feature request. To me, it makes no sense to show a tooltip when the lifecycle owner is paused. So either automatically dismiss the tooltip when receiving `Lifecycle.Event.ON_PAUSE` or add a new method:

`balloon.setAutoDismissWhenPaused()`

**Describe alternatives you've considered:**

Sure I can keep the balloon as a member variable and dismiss it explicitly in `onPause()`. It's ugly, and many developers will miss out on this hack.

**Comment by skydoves** [comment]
Hi, @znakeeye!
Thank you for your suggestion.
`setDismissWhenLifecycleOnPause` funtionality will be included in the next release.

**Comment by skydoves** [comment]
Hi, @znakeeye.
It is included in the new version `1.2.0`.
You can use the below methods together.
```kotlin
.setLifecycleOwner(value: LifecycleOwner)
.setDismissWhenLifecycleOnPause(value: Boolean)
```
